misc/discoverynode/src/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+echo hello from entry point
+TIMESTAMP=$(date +%s)
```

Comment:
Usually I use `date -Is -u` to give a standardized format:

`2024-10-13T16:06:56+00:00`